crates/external_services/Cargo.toml

```diff
@@ -31,14 +32,23 @@ hyper-proxy = "0.9.1"
 once_cell = "1.19.0"
 serde = { version = "1.0.197", features = ["derive"] }
 thiserror = "1.0.58"
-tokio = "1.37.0"
 vaultrs = { version = "0.7.2", optional = true }
+prost = "0.13"
+tokio = { version = "1.37.0", features = ["macros", "rt-multi-thread"] }
+tonic = "0.12.2"
+tonic-reflection = "0.12.2"
+tonic-types = "0.12.2"
```

Comment:
Can we make these dependencies optional, only pulling them in when the relevant gRPC features are enabled